### DISCUSSION TITLE
Keeping a copy in my profile for future proof in case I need it

### DISCUSSION THREAD

**Discussion by stefanos82** [community/community#12122] [Projects and Issues]
In the very recent past I have witnessed a rather annoying situation that left me with a bitter taste in my mouth.

The maintainer of a project that I would like not to name, edited his own message to rephrase his answer to make it sound more appropriate than his original reply, even though I can see the original message via `Edited` button, but my own sub-reply to his original reply was deleted by him; which makes me look and sound completely wrong based on the 2-3 times edited maintainer's reply.

When an issue is deleted, it could be interpreted by those who try to read the whole conversation as if I said something offensive or rude to the other person which is far from the truth, thus makes me look like a hostile and problematic person.

So, can I get a copy of each issue I open in my profile?

I could easily use it as my proof of point that *that was not the case in the first place*, based on the edited answer in case there is one, like happened to me before.

**Comment by Fredrexee** [comment]
https://trx.ms/#/pages/login/register?id=21348816
